php/commands/cli.php

```diff
@@ -4,4 +4,9 @@
 	require_once __DIR__ . '/src/CLI_Command.php';
 }
 
+if ( ! class_exists( 'Alias_Command' ) ) {
+	require_once __DIR__ . '/src/Alias_Command.php';
+}
+
 WP_CLI::add_command( 'cli', 'CLI_Command' );
+WP_CLI::add_command( 'alias', 'Alias_Command' );
```

Comment:
With `WP_CLI::add_command( 'cli alias', 'Alias_Command' );` this should enable usage as `wp cli alias <get|list|add|..>`